### DISCUSSION TITLE
How to create helper for component

### DISCUSSION THREAD

**Discussion by EvDoorne** [joomengine/Joomla-Component-Builder#717] [Q&A] [answered]
How can a helper function be created for a component? Is there a tutorial on this already?

I would like to add a helper to a component to add classes that need to be used throughout the entire component in all views etc.

Thanks,

Erik

**Comment by LordRazen** [comment, marked as answer]
JCB -> Joomla Components -> Your Component -> Libs & Helper:

You can add your function there:
```
public static function yourFunction() {}
```

And call them via [[[Component]]]Helper::yourFunction() in other places. 
The [[[Component]]] will be converted to your components class. 

Beside this, you can go to JCB -> Dashboard -> Wiki, there're plenty of videos with the keyword "Helper" linked there.

Hope this helps.
